protoc-gen-kotlin/macos/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("multiplatform")
```

Comment:
I did add the macos version of `protoc-gen-kotlin` but now sure it's 100% relevant. For now, it's not working because of the bug between kotlin multiplatform plugin and application plugin. I can just remove this if not require or create the workaround I did when checking the project structure.

Reply:
I don't think we really need it for now. The JVM version of `protoc-gen-kotlin` can be used from JS and Native builds without a problem.

Reply:
Removed